### ISSUE TITLE
npm WARN

### ISSUE THREAD

**Issue by MalgorzataK** [jacekokrojek/jak-to-zrobic-w-js#1]
Przy instalacji pakietów dostaję ostrzeżenie: "npm WARN package.json jak-to-zrobic-w-js@0.1.0 license should be a valid SPDX license expression".

**Comment by jacekokrojek** [comment]
Witam Pani Małgorzato, 
czy mimo komunikatu udało się uruchomić przykładowy test ?

**Comment by MalgorzataK** [comment]
Tak, udało mi się uruchomić przykładowy test. Chciałam tylko upewnić się czy mam ignorować ostrzeżenie przy instalacji pakietów?
